lib/util/git/types.ts

```diff
@@ -71,13 +71,20 @@ export interface FileDeletion {
 export type FileChange = FileAddition | FileDeletion;
 
 export interface CommitFilesConfig {
+  targetBranch?: string;
```

Comment:
this name is misleading, it's not the target branch to push to, it's the base branch to create the branch from and later merge to.